src/tests/functional/plugin/shared/include/behavior/ov_plugin/core_integration.hpp

```diff
@@ -373,6 +374,17 @@ TEST_P(OVClassSetModelPriorityConfigTest, SetConfigNoThrow) {
     EXPECT_EQ(value, ov::hint::Priority::HIGH);
 }
 
+TEST_P(OVClassSetAllowAutoBatchingConfigTest, SetConfigNoThrow) {
+    ov::Core ie = createCoreWithTemplate();
+    bool value = false;
+    OV_ASSERT_NO_THROW(ie.set_property(deviceName, ov::hint::allow_auto_batching(true)));
```

Comment:
test LoadNetwork() and compile_model() first

Reply:
to test LoadNetwork/compile_model need a real device like MULTI:CPU, so this seems to be a seperated  OVClass**CompileWith**AllowAutoBatchingConfigTest test instantiated accordingly
, actually the LoadNetwork is 1.0 API and should be validated in the corresponding **LoadWith**AllowAutoBatchingConfigTest test

Reply:
Why do you add tests for auto batch only? there are the same tests for other configs. It's just config and nothing other. Please check the functionality using the existed tests: to instantiate them for auto batch with required parameters (configs). The same comment for OpenVINO 2.0 API

Reply:
good point: the idea is that you can re-use existng tests (that check that setting some config value doesn't throw) just add {ALLOW_AUTO_BATCH, YES} and {ALLOW_AUTO_BATCH, NO} to the list of the configs e.g.    https://github.com/openvinotoolkit/openvino/blob/master/src/tests/functional/plugin/gpu/shared_tests_instances/behavior/executable_network/exec_net_base.cpp#L46